Example/Core/Tests/FIRComponentContainerTest.m

```diff
@@ -55,7 +55,7 @@ - (void)testRegisteringNonConformingClass {
 }
 
 - (void)testComponentsPopulatedOnInit {
-  FIRComponentContainer *container = [self containerWithRegistrants:@[ [FIRTestClass class] ]];
+  FIRComponentContainer *container = [self containerWithRegistrants:@ [[FIRTestClass class]]];
```

Comment:
This seems pretty broken. Does this still work as far as the compiler is concerned?

Maybe pull out a local reference to the class to make this format right?

However that solution stinks in `testRemoveAllCachedInstances`.

Reply:
Hrm, yeah...  but I'm not very interested in rewriting code in order to work around current clang-format annoyances.  I think we should either live with the annoyances (they'll likely go away in a future clang-format update?) or downgrade travis.  I am fine with either, though I don't know how to do the latter.

Reply:
(Oh, and yes, the code still compiles and passes tests)

Reply:
We can control which version of clang-format we install to the travis instance.  See https://github.com/firebase/firebase-ios-sdk/blob/master/.travis.yml#L15. 

But I'm in favor of upgrading to stay current and minimize onboarding hurdles.